ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/config_management_service.proto

```diff
@@ -18,6 +18,7 @@ import "google/protobuf/timestamp.proto";
 // The ledger configuration management service provides methods for the ledger administrator
 // to change the current ledger configuration. The services provides methods to modify
 // different aspects of the configuration.
+// This service is deprecated and will be removed in V2 Ledger API
```

Comment:
Thanks, really like the idea providing directions here :+1: 
@rgugliel-da is this precise? Can we set the maxdeduptime configuration overt the Canton admin API? I thought this is something which is a persisted (immutable) canton config :thinking:

Reply:
Correct. You cannot modify the max dedup time after the participant had been started for the first time. Eventually, however, we should make it modifyable - but that's way down on your priority list. So for Daml 3.0, I guess it will be immutable.

Reply:
As discussed offline, we go with "In V2 Ledger API this service is not available anymore."